corehq/sql_accessors/sql_templates/soft_undelete_forms.sql

```diff
@@ -0,0 +1,19 @@
+DROP FUNCTION IF EXISTS soft_undelete_forms(TEXT, TEXT[], TEXT);
+
+CREATE FUNCTION soft_undelete_forms(
+    p_domain TEXT,
+    form_ids TEXT[],
+    p_problem TEXT,
```

Comment:
do you mind if i do a follow up on that? i think it's a slightly larger change and would have to make sure we don't assume anywhere that Operations are just archiving/unarchiving. i know amy needs this fix